lib/actions/promote.coffee

```diff
@@ -0,0 +1,65 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+module.exports =
+	signature: 'promote [deviceIp]'
+	description: 'Promote a resinOS device'
+	help: '''
+		If you're running Windows, this command only supports `cmd.exe`.
```

Comment:
This is something that needs to be fixed in `rdt ssh` as well (where the message was picked from), I recently updated the documentation for `resin sync/ssh` and `rdt push` and this slipped through